backend/middlewares/middlewares.js

```diff
@@ -181,12 +181,27 @@
 		}
 	}
 
+	function isHereEnabled(req, res, next) {
+		const user = (req.session.user) ? req.session.user.username : "";
+		return User.isHereEnabled(user).then((hereEnabled) => {
```

Comment:
Are you checking the user who is logged to see if they have access to here maps? it should be the teamspace...

i.e If my account isn't here enabled, but TRL account is, I should be able to get HERE maps on TRL models, but not my models within my own account.